modules/apps/style-book/style-book-test/src/testIntegration/java/com/liferay/style/book/service/test/StyleBookEntryServiceTest.java

```diff
@@ -151,16 +194,7 @@ public void testGetStyleBookEntryByExternalReferenceCodeWithoutViewPermission()
 				RandomTestUtil.randomString(), null,
 				RandomTestUtil.randomString(), _serviceContext);
 
-		RoleTestUtil.removeResourcePermission(
-			RoleConstants.GUEST, StyleBookEntry.class.getName(),
-			ResourceConstants.SCOPE_INDIVIDUAL,
-			String.valueOf(styleBookEntry.getStyleBookEntryId()),
-			ActionKeys.VIEW);
-		RoleTestUtil.removeResourcePermission(
-			RoleConstants.SITE_MEMBER, StyleBookEntry.class.getName(),
-			ResourceConstants.SCOPE_INDIVIDUAL,
-			String.valueOf(styleBookEntry.getStyleBookEntryId()),
-			ActionKeys.VIEW);
+		_removeResourcePermissions(styleBookEntry.getStyleBookEntryId());
```

Comment:
Usually if there is only one usage, we keep it into the test method, but since it was extracted to a method, could you also apply it in `testGetStyleBookEntryByExternalReferenceCodeWithoutViewPermission` ? It seems same code is also there.

Reply:
Oh, I just see your next commit about improve performance. Since your new test was already with this improvement, to make easier to understand the flow, re-order the commit :slightly_smiling_face: